core/core-impl/src/main/java/io/fabric8/launcher/core/impl/documentation/BoosterReadmeProcessorImpl.java

```diff
@@ -0,0 +1,74 @@
+package io.fabric8.launcher.core.impl.documentation;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import io.fabric8.launcher.booster.catalog.rhoar.Mission;
+import io.fabric8.launcher.booster.catalog.rhoar.Runtime;
+import io.fabric8.launcher.core.api.documentation.BoosterDocumentationStore;
+import io.fabric8.launcher.core.api.readme.ReadmeProcessor;
+import org.apache.commons.lang3.text.StrSubstitutor;
+
+import static io.fabric8.launcher.core.impl.documentation.BoosterReadmePaths.getReadmePropertiesPath;
+import static io.fabric8.launcher.core.impl.documentation.BoosterReadmePaths.getReadmeTemplatePath;
+import static io.fabric8.launcher.core.impl.documentation.BoosterReadmePaths.loadContents;
+
+@ApplicationScoped
+public class BoosterReadmeProcessorImpl implements ReadmeProcessor {
+
+    private final BoosterDocumentationStore boosterDocumentationStore;
+
+    @Inject
+    public BoosterReadmeProcessorImpl(final BoosterDocumentationStore boosterDocumentationStore) {
+        this.boosterDocumentationStore = boosterDocumentationStore;
+    }
+
+    @Override
+    public String getReadmeTemplate(final Mission mission) throws IOException {
+        final Path path = getReadmeTemplatePath(getDocumentationBasePath(), mission.getId());
+        return loadContents(path);
+    }
+
+    @Override
+    @SuppressWarnings("all")
+    public Map<String, String> getRuntimeProperties(final String deploymentType, final Mission mission, final Runtime runtime) throws IOException {
+        Properties props = new Properties();
+
+        final Path path = getReadmePropertiesPath(getDocumentationBasePath(), deploymentType, mission.getId(), runtime.getId());
+
+        try (InputStream is = Files.newInputStream(path)) {
+            props.load(is);
+        }
+
+        Map<String, String> map = (Map) props;
```

Comment:
Add the SuppressWarning annotation in the code (see the Effective Java, page 124 - Eliminate Unchecked Warnings)

Reply:
Yup it is part of the rebase with master I was out of date